app/src/main/resources/db/sql/nakadi-db.sql

```diff
@@ -0,0 +1,91 @@
+CREATE SCHEMA IF NOT EXISTS zn_data AUTHORIZATION zalando_nakadi_data_owner;
```

Comment:
isn't it possible to trigger the scripts here directly?

Reply:
We will address this in a separate task. https://jira.wiley.com/browse/EPDCS-1023